config.toml

```diff
@@ -0,0 +1,21 @@
+[performance]
+par = 1
+dbcache = 1536
+blocksonly = 1
+maxuploadtarget = 137
+mempoolexpiry = 72
+maxmempool = 300
+maxorphantx = 100
+
+[spam_protection]
+limitfreerelay = 10
+minrelaytxfee = 0.0001
```

Comment:
If we can add to start options related to credentials, like `clientmustpaypublication=1`. Though for the other names, we can land at it is I’ll address them on their own.

Reply:
I would be happy to work on these. Ill write some docs on how to add something to the config and parser code too so it can be simple to do.